api/proxy/store/generated_key/memstore/ephemeraldb/ephemeral_db.go

```diff
@@ -68,14 +75,17 @@ func (db *DB) InsertEntry(key []byte, value []byte) error {
 
 	strKey := string(key)
 
+	derivationError := db.config.GetPUTWithGetReturnsDerivationError()
+
+	// disallow any overwrite
```

Comment:
ah I see this is unique to each entry - nvm plz ignore the aforementioned comment